.gitattributes

```diff
@@ -1,4 +1,9 @@
+.phpstan/ export-ignore
 .* export-ignore
 *.md export-ignore
 tests export-ignore
 docs export-ignore
+phpstan.neon.dist export-ignore
+phpstan-baseline.neon export-ignore
```

Comment:
If the .phpstan folder contains all the things, this should be removed right?

Reply:
Yes and no at the same time.

Yes because it does not contain those files

No because, 
1) people ALWAYS forgot to add those files to `export-ignore`
2) 3.x version will not live that long ( a lot of issues that are planned to be done in next major version ). In next major there will be only 2 files that will be moved to the root.

Based on those I've decided to leave it as it is now.